config/api.yml

```diff
@@ -621,6 +621,12 @@
         :identifier: storage_tag
       - :name: unassign
         :identifier: storage_tag
+  :ems_events:
```

Comment:
this is attempting to expose a new top level collection on the API /api/ems_events. Is that the intent ? 

Either way, I wouldn't do this here, furthermore, a provider specific subcollection might be more appropriate, /api/providers/:id/events or something like this.